extra-admin/bakka/autobuild/beyond

```diff
@@ -0,0 +1,3 @@
+abinfo "Installing bash script ..."
+install -Dvm644 "$SRCDIR"/shell/bakka.bash \
```

Comment:
Not how it works. You need to put this file somewhere, and source it in an /etc/profile.d script.